examples/pennylane_run_universal_classifier.py

```diff
@@ -317,7 +319,8 @@ def test(weights, x, y, state_labels=None):
     output_states = []
     dm_labels = [density_matrix(s) for s in state_labels]
     for i in range(len(x)):
-        expectation = qcircuit(weights, x=x[i], y=dm_labels[y[i]])
+        zvals = params[0] + params[1]*x[i]
```

Comment:
Just doing a fly-by review at the moment, but this caught my eye. What led to this reparameterization?

Reply:
This is the one that the paper uses. The basic idea is to introduce data and weights in the circuit using unitaries. So, instead of having a data unitary and then a weight unitary, they combine it in this way and apply it together. @mariaschuld mentioned this in a previous review. I am not sure if I want to completely following their implementation word by word though. Eg., they use a weighted fidelity function as los and here I am not using that.